packages/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -40,14 +41,15 @@ const DataGridPremiumRaw = React.forwardRef(function DataGridPremium<R extends G
 ) {
   const props = useDataGridPremiumProps(inProps);
   const privateApiRef = useDataGridPremiumComponent(props.apiRef, props);
-
+  const hooks = { useGridRowAriaAttributes };
+  const configuration = { hooks };
```

Comment:
These need to be defined outside of the component. Because they're objects they will trigger a re-render of every useContext otherwise.

Reply:
Updated, thanks 🙂